rust/chains/hyperlane-sealevel/src/mailbox.rs

```diff
@@ -1,62 +1,70 @@
+use hyperlane_sealevel_mailbox::instruction::InboxProcess;
+use hyperlane_sealevel_mailbox::{
+    mailbox_dispatched_message_pda_seeds, mailbox_inbox_pda_seeds, mailbox_outbox_pda_seeds,
+    mailbox_process_authority_pda_seeds, mailbox_processed_message_pda_seeds,
+};
+use hyperlane_sealevel_message_recipient_interface::{
+    HandleInstruction, MessageRecipientInstruction,
+};
+use serializable_account_meta::SimulationReturnData;
+use solana_account_decoder::{UiAccountEncoding, UiDataSliceConfig};
+use solana_client::{
+    nonblocking::rpc_client::RpcClient,
+    rpc_config::{RpcAccountInfoConfig, RpcProgramAccountsConfig, RpcSendTransactionConfig},
+    rpc_filter::{Memcmp, MemcmpEncodedBytes, RpcFilterType},
+};
+use solana_sdk::compute_budget::ComputeBudgetInstruction;
+use solana_sdk::instruction::Instruction;
+use solana_sdk::{
+    account::Account,
+    commitment_config::CommitmentConfig,
+    hash::Hash,
+    instruction::AccountMeta,
+    message::Message,
+    pubkey::Pubkey,
+    signature::Signature,
+    signer::{keypair::Keypair, Signer as _},
+    transaction::{Transaction, VersionedTransaction},
+};
+use solana_transaction_status::{
+    EncodedConfirmedBlock, EncodedTransaction, EncodedTransactionWithStatusMeta,
+    UiInnerInstructions, UiInstruction, UiMessage, UiParsedInstruction, UiReturnDataEncoding,
+    UiTransaction, UiTransactionReturnData, UiTransactionStatusMeta,
+};
 
+use crate::RpcClientWithDebug;
 use crate::{
-    mailbox::contract::DispatchedMessageAccount,
-    mailbox_inbox_pda_seeds, mailbox_message_storage_pda_seeds, mailbox_outbox_pda_seeds,
-    mailbox_process_authority_pda_seeds, mailbox_processed_message_pda_seeds,
-    solana::{
-        account::Account,
-        account_decoder::{UiAccountEncoding, UiDataSliceConfig},
-        commitment_config::CommitmentConfig,
-        hash::Hash,
-        instruction::{AccountMeta, Instruction},
-        message::Message,
-        nonblocking_rpc_client::RpcClient,
-        pubkey::Pubkey,
-        rpc_config::{RpcAccountInfoConfig, RpcProgramAccountsConfig, RpcSendTransactionConfig},
-        rpc_filter::{Memcmp, MemcmpEncodedBytes, RpcFilterType},
-        signature::Signature,
-        signer::{keypair::Keypair, Signer as _},
-        transaction::{Transaction, VersionedTransaction},
-        transaction_status::{
-            EncodedConfirmedBlock, EncodedTransaction, EncodedTransactionWithStatusMeta,
-            UiInnerInstructions, UiInstruction, UiMessage, UiParsedInstruction,
-            UiReturnDataEncoding, UiTransaction, UiTransactionReturnData, UiTransactionStatusMeta,
-        },
-    },
     utils::{get_account_metas, simulate_instruction},
     ConnectionConf, SealevelProvider,
 };
 
-use crate::RpcClientWithDebug;
-
-use self::contract::{
-    SerializableAccountMeta, SimulationReturnData, DISPATCHED_MESSAGE_DISCRIMINATOR,
-};
+const SYSTEM_PROGRAM: &str = "11111111111111111111111111111111";
```

Comment:
I recently learned about this macro https://docs.rs/solana-program/latest/solana_program/pubkey/index.html that lets you do:
```
const SYSTEM_PROGRAM: Pubkey = pubkey!("11111111111111111111111111111111");
```
May as well move over to that here while you're here so we don't have to do the `Pubkey::from_str`s in a few places